<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-12QPG602" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="30 Apr 2021 07:09:08 GMT" time="0.21" errors="0">
  <testcase name="testcase2" time="0.038" classname="testngpractise.TestClass">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testngpractise.TestClass.testcase2(TestClass.java:20)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase3" time="0.0" classname="testngpractise.TestClass">
    <skipped/>
  </testcase> <!-- testcase3 -->
  <testcase name="testcase1" time="0.0" classname="testngpractise.TestClass">
    <ignored/>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Default test(failed) -->
